front-end/src/components/Post/SecondProposal.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useContext } from 'react';
+import { /* Divider, */ Grid } from 'semantic-ui-react';
+import styled from '@xstyled/styled-components';
+import { ApiPromise } from '@polkadot/api';
+import { InjectedAccountWithMeta } from '@polkadot/extension-inject/types';
+
+import { Form } from '../../ui-components/Form';
+import Button from '../../ui-components/Button';
+import { NotificationContext } from '../../context/NotificationContext';
+import { NotificationStatus } from '../../types';
+
+interface Props {
+	className?: string
+	proposalId?: number | null | undefined
+	api?: ApiPromise,
+	apiReady?: boolean,
+	getLinkedAccount: () => Promise<InjectedAccountWithMeta | undefined>
+}
+
+const SecondProposal = ({ className, api, apiReady, getLinkedAccount, proposalId }: Props) => {
+	const { queueNotification } = useContext(NotificationContext);
+
+	const secondProposal = async () => {
+		if (!api) {
+			return;
+		}
+
+		const linkedAccount = await getLinkedAccount();
+
+		if (!linkedAccount) {
+			return;
+		}
+
+		const second = api.tx.democracy.second(proposalId || 0);
```

Comment:
We should bail out if there's no proposalId. Defaulting to 0 won't bring users much further and might be hard to debug down the road.

Reply:
done